testsuite/MDAnalysisTests/coordinates/data/create_data.py

```diff
@@ -0,0 +1,39 @@
+import MDAnalysis as mda
+import numpy as np
+from six.moves import range
+
+
+def create_test_trj(uni, fname, wrong_natoms=False):
+    n_atoms = uni.atoms.n_atom
+    if wrong_natoms:
+        n_atoms -= 2
+    pos = np.arange(3 * n_atoms).reshape(n_atoms, 3)
+    uni.trajectory.ts.dt = 1
+    with mda.Writer(fname, n_atoms) as w:
```

Comment:
I understand your concern. But i don't think just generating files with another program is the right solution. The other problem could also have a bug that we undo transparently in the reader. 

Instead I suggest another procedure to generate the test files. We will generate them with MDAnalysis, because it is convenient and we can be confident that it works since we don't start from scratch. Afterwards we check the content of the files with another tool, or several tools. This means we validate the files as best as we can instead of hoping the other tools don't have bugs.

Files would also be only generated rarely. But i thought I should add the scripts so that in time people will know how they have been generated. I will also include verification scripts for the files that use external tools. But I'm not very familiar with other stuff (VMD scripting, gromacs tools, NAMD) so i appreciate some help with the validation scripts.